scripts/nvcc

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# (C) Copyright IBM Corp. 2018, 2020. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"${CUDA_HOME}/bin/nvcc" -ccbin "${CXX}" $@
```

Comment:
I'm worried this might cause a problem with some of our builds. We spent some time adding the `-ccbin` option to our build scripts. I like this solution better, but I think I've seen it throw an error before when `-ccbin` is specified twice. Will need to keep a look out for this. I think it makes sense to keep this in for now, but when we are switching our recipes over to work with this new packages, keep in mind we might have to modify them to get around this.

Reply:
Good point. using `ccbin` is pretty common, but do we bake it in or leave it to the various other recipes to manage and have this script be a true pass through.
We'll likely be changing every `build.sh` anyway so we can see how it goes.